### PR TITLE
feat: sync development to staging

### PR DESCRIPTION
**Pull request by grikomsn**
## Description

This PR syncs latest development branch to staging.

## Changes

- [x] https://github.com/CosmosContracts/juno-tools/pull/22
- [x] https://github.com/CosmosContracts/juno-tools/pull/23
- [x] https://github.com/CosmosContracts/juno-tools/pull/24
- [x] https://github.com/CosmosContracts/juno-tools/pull/25
- [x] https://github.com/CosmosContracts/juno-tools/pull/29
- [x] https://github.com/CosmosContracts/juno-tools/pull/30
- [x] https://github.com/CosmosContracts/juno-tools/pull/32
- [x] https://github.com/CosmosContracts/juno-tools/pull/33
- [x] https://github.com/CosmosContracts/juno-tools/pull/34
- [x] https://github.com/CosmosContracts/juno-tools/pull/39
- [x] https://github.com/CosmosContracts/juno-tools/pull/40
- [x] https://github.com/CosmosContracts/juno-tools/pull/41
- [x] https://github.com/CosmosContracts/juno-tools/pull/42
- [x] https://github.com/CosmosContracts/juno-tools/pull/43
- [x] https://github.com/CosmosContracts/juno-tools/pull/44

## Screenshots

\-

## Testing Steps

\-

## Links

\-

### CODE REVIEW

**Comment by grikomsn** [comment]
Assuming this is merged, next PR would be merging `staging` to `main`, right?

cc @findolor @RyanWarner

**Comment by RyanWarner** [comment]
@grikomsn Please don't merge this without explicit approval from @findolor and @Highlander-maker.

As far as a production deploy, let's let @findolor drive that. We can let them know our work is ready, but let's let the Juno devs make that PR when they feel comfortable. We don't have enough context on their work and commitments to make that decision.

**Comment by findolor** [comment]
@grikomsn @RyanWarner @XanderJL  amazing work guys! A lot has changed since I was sleeping ahah. It's really nice to see the project coming along in such a good way.

As for the merge let us plan things a bit and I'll keep you updated on how we are going the proceed.

File: components/Sidebar.tsx [hunk 1]
```diff
@@ -61,11 +61,11 @@ const Sidebar = () => {
 
       {/* footer attribution */}
       <div className="text-xs text-white/50">
-        JunoTools 1.0. Copyright &copy; {new Date().getFullYear()} by{' '}
+        JunoTools 0.1.0-beta <br />
```
Comment (grikomsn) [inline components/Sidebar.tsx:65]: How about getting the version from `package.json`?

Comment (RyanWarner) [reply components/Sidebar.tsx:65]: I like this idea @grikomsn — any ideas how we would accomplish that?

Comment (grikomsn) [reply components/Sidebar.tsx:65]: - https://github.com/CosmosContracts/juno-tools/pull/47

**Pull request merged**